mem0/graphs/utils.py

```diff
@@ -37,6 +37,7 @@
 1. Extract only explicitly stated information from the text.
 2. Identify nodes (entities/concepts), their types, and relationships.
 3. Use "USER_ID" as the source node for any self-references (I, me, my, etc.) in user messages.
+CUSTOM_PROMPT
```

Comment:
Shouldn't this be inside a {} ?

Reply:
I am using .replace() here, so it just replaces the string directly

Reply:
Sounds good.